### PR TITLE
Md docker cronfix

### PR DESCRIPTION
**Pull request by MaximeDavin**
Fixed for croncheck.sh

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -85,14 +85,12 @@
...
-    hour: "{{ item.schedule[1] }}"
-    day: "{{ item.schedule[2] }}"
-    month: "{{ item.schedule[3] }}"
-    weekday: "{{ item.schedule[4] }}"
-  with_items: "{{ docker_cron_tasks }}"
-  when: docker_cron_tasks
+      name: "Docker prune"
+      user: root
+      special_time: "weekly"
+      job: docker system prune -f
```
Comment (MaximeDavin) [inline tasks/main.yml]: Feel free to silence the cron with >/dev/null if you think that we do not need the cron's output mail.

File: tasks/main.yml [hunk 2]
```diff
@@ -85,14 +85,12 @@
...
-    minute: "{{ item.schedule[0] }}"
-    hour: "{{ item.schedule[1] }}"
-    day: "{{ item.schedule[2] }}"
-    month: "{{ item.schedule[3] }}"
-    weekday: "{{ item.schedule[4] }}"
-  with_items: "{{ docker_cron_tasks }}"
-  when: docker_cron_tasks
+      name: "Docker prune"
+      user: root
+      special_time: "weekly"
```
Comment (MaximeDavin) [inline tasks/main.yml]: or monthly

File: tasks/main.yml [hunk 3]
```diff
@@ -85,14 +85,12 @@
     mode: "0755"
   when: docker_install_docker_compose
 
-- name: Create cron tasks for Docker commands
+- name: Cron Docker clean up 
   cron:
-    name: "{{ item.name }}"
```
Comment (saraELAFIA) [inline tasks/main.yml]: we could keep the item logic and simply add a cron_file entry in each item 
-docker_cron_tasks:
- job: docker system prune -f
    name: "Docker clean up"
    schedule: ["0", "0", "*", "*", "0"]
    cron_file: [here]

File: defaults/main.yml [hunk 4]
```diff
@@ -9,11 +9,6 @@ docker_compose_version: "1.19.0"
 
 docker_users: []
 
-docker_cron_tasks:
```
Comment (saraELAFIA) [inline defaults/main.yml]: we could keep the item logic and simply add a cron_file entry in each item 
-docker_cron_tasks:
- job: docker system prune -f
    name: "Docker clean up"
    schedule: ["0", "0", "*", "*", "0"]
    cron_file: [here]

File: tasks/main.yml [hunk 5]
```diff
@@ -85,14 +85,12 @@
...
-    job: "{{ item.job }}"
-    minute: "{{ item.schedule[0] }}"
-    hour: "{{ item.schedule[1] }}"
-    day: "{{ item.schedule[2] }}"
-    month: "{{ item.schedule[3] }}"
-    weekday: "{{ item.schedule[4] }}"
```
Comment (saraELAFIA) [inline tasks/main.yml]: cron_file: {{item.cron_file}}

File: tasks/main.yml (hunk 1 above)
Comment (saraELAFIA) [reply tasks/main.yml]: yes good idea, let's add it

File: defaults/main.yml (hunk 4 above)
Comment (MaximeDavin) [reply defaults/main.yml]: So which one do we keep ? the 'with_items' version ?

Comment (saraELAFIA) [reply defaults/main.yml]: @MaximeDavin yes no need to change all the logic for just one argument cron_file :-)

**Pull request closed without merging**